microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -51,14 +143,19 @@ Key compatibility limitations include the operating system and rendering in fram
 <!-- ------------------------------ -->
 #### Operating systems
 
-Windows 7 and Windows 8 can only do windowed hosting, not visual hosting.
+All hosting modes are supported wherever WebView2 is supported ie. Windows 10 and up and certain Windows Server versions. Windows 7, 8 and 8.1 are no longer supported. For those still on those platforms Windows 7 and Windows 8 can only do Windowed hosting, not Visual hosting.
 
 See [Windows 7 and 8](../index.md#windows-7-and-8) in _Introduction to Microsoft Edge WebView2_.
 
 
 <!-- ------------------------------ -->
 #### Rendering WebView2 in framework and non-framework apps
 
+<!-- todo: is this tabset/ APIs list introduced ok?  is this section a straight copy of "Overview of APIs"?  link to there instead?  
```

Comment:
is this tabset/ APIs list introduced ok? is this section a straight copy of "Overview of APIs"? link to there instead?

that link would be:
* [Rendering WebView2 in non-framework apps](./overview-features-apis.md#rendering-webview2-in-non-framework-apps) - what about "in framework"??

Present article:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/windowed-vs-visual-hosting?branch=pr-en-us-3271&tabs=dotnetcsharp#rendering-webview2-in-framework-and-non-framework-apps
Overview APIs:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3271&tabs=dotnetcsharp#rendering-webview2-in-non-framework-apps

Reply:
in every API section in article, linked to same/similar/equivalent section in "Overview of APIs" article.  for maintainability; we can easily compare the two sections to see if same, and why heading sometimes differs.